libs/ui/src/lib/benchmark/benchmark.component.html

```diff
@@ -6,6 +6,31 @@
     </td>
   </ng-container>
 
+  <ng-container matColumnDef="date">
+    <th
+      *matHeaderCellDef
+      class="d-none d-lg-table-cell text-right"
+      mat-header-cell
+    >
+      <span class="d-none d-sm-block text-nowrap" i18n>Last All Time High</span>
+    </th>
+    <td
+      *matCellDef="let element"
+      class="d-none d-lg-table-cell text-right"
+      mat-cell
+    >
+      <gf-value
+        class="d-inline-block justify-content-end text-primary"
```

Comment:
Can you remove class here and wrap `gf-value` in a `<div class="d-flex justify-content-end">`?